includes/class-capability.php

```diff
@@ -5,10 +5,21 @@
 final class Capability {
 	const MANAGE_REDIRECTS_CAPABILITY = 'manage_redirects';
 
+	// Used to flip the version of the available roles capabilities.
+	const CAPABILITIES_VER = '1';
```

Comment:
This is still a string? It's used in line 19 with a `<=` operator.

Reply:
Fixed now